scalding-parquet-scrooge/src/main/java/com/twitter/scalding/parquet/scrooge/ParquetScroogeOutputFormat.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2012 Twitter, Inc.
```

Comment:
Yeah, but we *do* probably need to leave the license headers as-is from files that we are copying from apache projects